src/Kiota.Builder/Writers/Php/CodeMethodWriter.cs

```diff
@@ -63,6 +67,9 @@ public override void  WriteCodeElement(CodeMethod codeElement, LanguageWriter wr
                     case CodeMethodKind.RequestExecutor:
                         WriteRequestExecutorBody(codeElement, parentClass, requestParams, writer);
                         break;
+                    case CodeMethodKind.Factory when codeElement.IsOverload:
+                        WriteFactoryMethodOverloadBody(codeElement, parentClass, writer);
+                        break;
```

Comment:
Since we are no longer genetating overloads in the refiner, should we remove this and the `WriteFactoryMethodOverloadBody` function and related variables?

Reply:
Make sense, updated.